internal/transform/transform.go

```diff
@@ -288,11 +288,6 @@ func ExtractScript(doc *astro.Node, n *astro.Node, opts *TransformOptions) {
 func AddComponentProps(doc *astro.Node, n *astro.Node, opts *TransformOptions) {
 	if n.Type == astro.ElementNode && (n.Component || n.CustomElement) {
 		for _, attr := range n.Attr {
-			id := n.Data
-			if n.CustomElement {
```

Comment:
this doesn't look related to this change, are you sure it's safe to remove this? I know we don't support using CE tag names any more but I think we still print them to go through renderComponent, right?

Reply:
Yep, thanks for checking! After the refactoring below, `id` is unused. We still handle CE tags here, but we're also handling Component tags the same way now (as a string)

Reply:
Oh got it, since we are not using metadata.getPath we don't need the id here at all. Thanks.